README.md

```diff
@@ -442,6 +442,30 @@ error.name           # => "address.street"
 error.attribute_name # => "street"
 ```
 
+Also you can use validations nested in Hanami Validations object.
```

Comment:
`Validations can be nested in Hanami::Validations objects`
^
statement

`Also you can use validations nested in Hanami Validations object`
^
action